### ISSUE TITLE
Show current exercises where no solution is submitted yet

### ISSUE THREAD

**Issue by SteffenBauer** [exercism/exercism.io#1972]
I returned to continue solving exercism problems after some days of inactivity. Before that I had fetched new exercises. It took some time of tedious searching to find the new exercises among the many exercises I had solved so far.

Proposal: Show also current exercises with zero iterations in the account overview, so one can see immediately which are the newest exercises one has fetched. Perhaps by introducing a new field "Fetched" into the "Exercises" overview in "Account", so that there are three fields "Fetched", "Current", "Completed".

**Comment by carlisia** [comment]
This issue is referenced on #137, can we close it?

**Comment by kytrinyx** [comment]
> Show also current exercises with zero iterations in the account overview

This is not the same suggestion. The suggestion in exercism/cli#137 is for a CLI command, whereas this is about putting something on the website.

That said, we do kind of show something like this on the dashboard now ("fetched"). There's some more work to do to enhance it, but I think we can close this and open new issues about improving the fetched/not-solved section.

**Comment by kytrinyx** [comment]
OK, there's been a bit of churn around the whole status/fetched/unsubmitted thing, and I spent some time taking a step back and thinking about how to solve it.

I found an approach that I liked, and then took a bigger step back and realized that all of this complexity is in the app kind of haphazardly. I've taken the slightly painful step of ripping it out (perhaps temporarily) to see what it feels like. Will people miss it? I don't know. Maybe.

I have some ideas about this, but I want to give it a few days to a few weeks to simmer.

I want to thank everyone who has worked on the features, posted the bugs, tweaked, thrashed, discussed, mused, and reflected. It's truly a pleasure to get your help figuring this out.

_cross-posted this comment in all the threads that are thrashing around with this feature, so that everyone involved knows what's going on_

:heart:
